docs/src/guide/vue-flow/config.md

```diff
@@ -652,14 +652,18 @@ const elements = ref([
 
 ### edges-updatable (optional)
 
-- Type: `boolean`
+- Type: `EdgeUpdatable`
 
 - Default: `true`
 
 - Details:
 
   Globally enable/disable updating edges.
 
+  If set to 'source' only source markers are updatable
+  
+  If set to 'target' only target markers are updatable
```

Comment:
Can you add a line saying if set to `true` it will enable it for both `source` and `target`?

Reply:
Sure thing. I will add it now!